rig/machine_control/bmp_controller.py

```diff
@@ -168,9 +168,8 @@ def _send_scp(self, cabinet, frame, board, *args, **kwargs):
 
         return connection.send_scp(length, 0, 0, board, *args, **kwargs)
 
-    @ContextMixin.use_contextual_arguments
-    def get_software_version(self, cabinet=Required, frame=Required,
-                             board=Required):
+    @ContextMixin.use_contextual_arguments()
```

Comment:
I dislike the mandatory brackets... but, if I remember correctly, this is a hard problem to fix.